src/db.js

```diff
@@ -26,7 +26,7 @@ const sequelize = new Sequelize({
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '123123',
```

Comment:
The default password '123123' is hardcoded. While this might be acceptable for development, ensure that this is replaced with a secure password in production environments. Consider using environment variables exclusively for sensitive information.